network/network-address/src/lib.rs

```diff
@@ -1007,8 +960,8 @@ mod test {
             ((&dns_name, 123), expected_suffix)
         );
 
-        let addr = NetworkAddress::from_str("/dns/example.com/tcp/123/ln-peerid-ex").unwrap();
-        let expected_suffix: &[Protocol] = &[Protocol::PeerIdExchange];
+        let addr = NetworkAddress::from_str("/dns/example.com/tcp/123/memory/44").unwrap();
```

Comment:
do we have any docs in this code that need to be updated?

Reply:
think i got everything!